ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorFunctionalTest.java

```diff
@@ -51,15 +51,20 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
+import org.junit.FixMethodOrder;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @Category({IntegrationTest.class})
 @RunWith(MockitoJUnitRunner.class)
+// shouldFailOnAvroWithoutSchemasIfSchemaNotEvolvable fails if run after shouldHandleJsonWithSchemas
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
tbh, this isn't great practice 😅 but I didn't want to distract from the fix. In general tests shouldn't depend on ordering because then you can't parallelize them. Hopefully I'll have time to debug why this is happening, but I spent like 20min and couldn't immediately figure it out